setup.cfg

```diff
@@ -43,5 +43,5 @@ multi_line_output = 3
 use_parentheses = True
 
 [flake8]
-ignore = E203, E501, E741, W503
+ignore = E203, E501, E741, W503, W605
```

Comment:
This is to be able to write LaTeX in docstrings. This category of errors is like "\s is not a valid escape character". I don't think it's too dangerous to ignore those compared to the benefit it adds, but I can revert and remove my formula if someone feels strongly against this.

Reply:
Pinging @LysandreJik so you're aware of that change, merging now but we can revert later if you feel strongly against it.

Reply:
I feel strongly about it, but in a good way :) It's a welcome addition!